application_form/api/serializers.py

```diff
@@ -141,6 +141,7 @@ class ApartmentReservationSerializerBase(serializers.ModelSerializer):
     class Meta:
         model = ApartmentReservation
         fields = (
+            "id",
```

Comment:
Is this something that we want to expose here? There is also the `apartment_uuid` that would be the "public" ID in this case. I'm not 100% sure how the logic went here, but at some point, we talked about only using those for the APIs, but that might have been for something else as well. Changing the lookup field for the viewset from `id` can be done with `lookup_field="apartment_uuid"` if you go that route.

If you go with the `apartment_uuid`, I suggest adding an `index=True` to that field in the model.

Reply:
Hmm, not sure what you mean. These are reservations, and there can be many of these related to an apartment, so we cannot refer to one of these with just an apartment UUID.

Reply:
Ah, damn. I must have been tired when reviewing. I mistake the reservation for just the apartment 😄  Sorry about this, no changes or suggestions here :)